src/openrpc/lifecycle_management.json

```diff
@@ -419,15 +402,15 @@
                             "sessionId": "c0d012b4-0854-4bbc-82d5-ab52ac29dd94",
```

Comment:
appId is missing in this response.

Reply:
it is there, did you expand the up in what github shows?